pages/resources/collectibles.mdx

```diff
@@ -0,0 +1,802 @@
+| ----------- | ------------------ | ------------------------- |
+| amount      | integer            | How much the discount is  |
+| discount_id | snowflake          | The discount ID           |
+| expires_at  | ?ISO8601 timestamp | When the discount expires |
+
+###### Gift Eligibility Structure
+
+| Field | Type    | Description                                           |
+| ----- | ------- | ----------------------------------------------------- |
+| valid | boolean | Whether the recipient is eligible to receive the gift |
+
+###### Variant Return Style
+
+| Value | Name                | Description                                        |
+| ----- | ------------------- | -------------------------------------------------- |
+| 1     | INDIVIDUAL_PRODUCTS | Variants should be returned as individual products |
+| 2     | VARIANTS_GROUP      | Variants should be returned as a group             |
+
+###### Collectibles Shop Home Config
+
+| Value                  | Name                   | Description                                        |
+| ---------------------- | ---------------------- | -------------------------------------------------- |
+| default                | DEFAULT                | The default shop home configuration                |
+| default_with_orb_shelf | DEFAULT_WITH_ORB_SHELF | The default shop home configuration with orb shelf |
+| orb_tab                | ORB_TAB                | The orb tab shop home configuration                |
+| summer_sale_takeover   | SUMMER_SALE_TAKEOVER   | The summer sale takeover shop home configuration   |
+
+###### Collectibles Shop Home Tab
+
+| Value   | Name    | Description     |
+| ------- | ------- | --------------- |
+| home    | HOME    | The home tab    |
+| catalog | CATALOG | The catalog tab |
+| orbs    | ORBS    | The orbs tab    |
+
+###### Shop Block Type
+
+| Value | Name             | Description                    |
+| ----- | ---------------- | ------------------------------ |
+| 0     | HERO             | A hero shop block              |
+| 1     | FEATURED         | A featured shop block          |
+| 2     | FEED             | A feed shop block              |
+| 3     | WIDE_BANNER      | A wide banner shop block       |
+| 4     | SHELF            | A shelf shop block             |
+| 5     | COUNTDOWN_TIMER  | A countdown timer shop block   |
+| 6     | IMMERSIVE_BANNER | An immersive banner shop block |
+
+###### Shop Item Types
+
+| Value             | Name              | Description                      |
+| ----------------- | ----------------- | -------------------------------- |
+| AVATAR_DECORATION | Avatar Decoration | An avatar decoration product     |
+| PROFILE_EFFECT    | Profile Effect    | A profile effect product         |
+| NAMEPLATE         | Nameplate         | A nameplate product              |
+| BUNDLE            | Bundle            | A bundle of collectible products |
+
+###### Shop Product Themes
```

Comment:
why is this in the marketing object (same as above)
also applies to all the below enums and structures, i don't wanna copy paste the same review again